README.md

```diff
@@ -7,8 +7,20 @@ Please note that extensive documentation on design can be found in the following
 * [Implementation Specification](https://docs.google.com/document/d/1wjg3hl0iDLNh7jIRaHl3IXhwm0ssOtDje5NemyTBcaw/edit?usp=sharing)
 * [Deployment Considerations](https://docs.google.com/document/d/17wOs-T7mugwte5_Dt-KLGMsp-3_yAARejpFmrAMefSE/edit?usp=sharing)
 
+
 # Instructions on use of the Uptane demonstration code
-## Installation
+
+Below are the instructions on use of the Uptane demonstration and reference
+implementation code, divided into these sections:
+
+* [0: Installation](#0-installation)
```

Comment:
Should the Table of Contents also include subheadings?  An outline of the demo's workflow and attacks might help the reader.